src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.html

```diff
@@ -1,10 +1,10 @@
 <ng-container *ngIf="(isArchived$ | async) === false">
-  <app-button *ngIf="!isEditing; else editing" id="edit" (clicked)="checkIfEditableReasonRequired()">Amend</app-button>
+  <app-button *ngIf="!isEditing; else editing" [id]="customId + 'edit'" (clicked)="checkIfEditableReasonRequired()">Amend</app-button>
```

Comment:
I personally think working customId's in is overkill. Just 'BottomEdit' I think would be fine? Also on this app-button idk if I am being silly and missing something but where is customId defined?

Reply:
It's defined in the `.ts` of the component?

Reply:
We want ids to be unique, there's two of those components of the page. How else would we get unique ids on that page?

Reply:
Yeah I was being silly! I gave myself the out 😂